htlcswitch/switch.go

```diff
@@ -559,8 +559,6 @@ func (s *Switch) CloseLink(chanPoint *wire.OutPoint,
 		return updateChan, errChan
 
 	case <-s.quit:
-		errChan <- errors.New("unable close channel link, htlc " +
```

Comment:
This will likely not be in the final version of the PR, I was trying to test using unbuffered channels on travis and didn't want the code to hang if it exited for whatever reason.  I will remove this after rebasing.